samples/azure/azure-container-instances-dab-starter.ps1

```diff
@@ -0,0 +1,705 @@
+    Write-ErrorMessage "DAB config file not found at: $DabConfigFile"
+    exit 1
+}
+
+Write-Success "Repository root: $repoPath"
+Write-Success "Using DAB config: $DabConfigFile"
+
+# Generate or validate parameters
+if (-not $ResourcePrefix) {
+    $ResourcePrefix = "dab$(Get-Random -Maximum 9999)"
+    Write-Host "Generated resource prefix: $ResourcePrefix" -ForegroundColor Yellow
+}
+
+if (-not $ResourceGroup) {
+    $ResourceGroup = "rg-$ResourcePrefix"
+    Write-Host "Generated resource group: $ResourceGroup" -ForegroundColor Yellow
+}
+
+# Set subscription
+if ($SubscriptionId) {
+    Write-Step "Setting Azure subscription..."
+    az account set --subscription $SubscriptionId
+    Write-Success "Subscription set to: $SubscriptionId"
+}
+else {
+    $currentSub = az account show | ConvertFrom-Json
+    $SubscriptionId = $currentSub.id
+    Write-Host "Using current subscription: $($currentSub.name) ($SubscriptionId)" -ForegroundColor Yellow
+}
+
+# Generate resource names
+$acrName = Get-UniqueResourceName -Prefix $ResourcePrefix -Type "acr"
+$acrName = $acrName -replace "[^a-zA-Z0-9]", "" # ACR names must be alphanumeric
+$aciName = Get-UniqueResourceName -Prefix $ResourcePrefix -Type "aci"
+$sqlServerName = Get-UniqueResourceName -Prefix $ResourcePrefix -Type "sql"
+$sqlDbName = "dabdb"
+$sqlAdminUser = "sqladmin"
+$dnsLabel = Get-UniqueResourceName -Prefix $ResourcePrefix -Type "dab"
+$dnsLabel = $dnsLabel -replace "[^a-zA-Z0-9-]", "" # DNS labels must be alphanumeric with hyphens
+$acrImageName = "dab"
+$acrImageTag = "latest"
+
+# Convert SecureString to plain text with proper memory management
+$bstr = [System.Runtime.InteropServices.Marshal]::SecureStringToBSTR($SqlAdminPassword)
+try {
+    $sqlAdminPasswordPlain = [System.Runtime.InteropServices.Marshal]::PtrToStringAuto($bstr)
+}
+finally {
+    [System.Runtime.InteropServices.Marshal]::ZeroFreeBSTR($bstr)
+}
+
+# Summary
+Write-Host "`nDeployment Configuration:" -ForegroundColor Cyan
+Write-Host "  Subscription:     $SubscriptionId"
+Write-Host "  Resource Group:   $ResourceGroup"
+Write-Host "  Location:         $Location"
+Write-Host "  ACR Name:         $acrName"
+Write-Host "  Container Name:   $aciName"
+Write-Host "  DNS Label:        $dnsLabel"
+Write-Host "  SQL Server:       $sqlServerName"
+Write-Host "  SQL Database:     $sqlDbName"
+Write-Host "  SQL Admin User:   $sqlAdminUser"
+Write-Host "  Container Port:   $ContainerPort"
+Write-Host "  Container CPU:    $ContainerCpu cores"
+Write-Host "  Container Memory: $ContainerMemory GB"
+
+$confirmation = Read-Host "`nProceed with deployment? (y/N)"
+if ($confirmation -ne 'y') {
+    Write-Host "Deployment cancelled." -ForegroundColor Yellow
+    exit 0
+}
+
+try {
+    # Create Resource Group
+    Write-Step "Creating resource group..."
+    az group create --name $ResourceGroup --location $Location | Out-Null
+    Write-Success "Resource group created: $ResourceGroup"
+
+    # Create ACR
```

Comment:
ACR creation is unnecessary. Same comments similar to the other ACA sample script